docs/design/datacontracts/Loader.md

```diff
@@ -15,10 +15,32 @@ readonly struct ModuleHandle
 [Flags]
 enum ModuleFlags
 {
+    Tenured = 0x00000001, // Set once we know for sure the Module will not be freed until the appdomain itself exits
```

Comment:
There needs to be a detail in the versioned portion of the contract indicating which module flags are actually handled by that specific version of the contract. I want to be able to know which magic constants are meaningful, and the current document does not explain. (This becomes relevant since we may renumber the enum in the runtime, but we won't be able to renumber the enum in the contract interface.)